server/core/config/valid/global_cfg.go

```diff
@@ -154,9 +154,17 @@ type BasicAuth struct {
 	Username string
 	Password string
 }
+
+type TaskQueue struct {
+	ActivitiesPerSecond string
```

Comment:
Why isn't this just a float? Each time you use this you're doing a conversion and that doesn't make sense.

Reply:
> We only use it once for each task queue when setting up the worker process.
Thats the only time its every used so would the value ever need to be a string.

> I initially defined this as a float but realized it might not align with what we currently have in place since the ToValid() call does not return any errors.
I think you need to read up on our validation library because that should validate numbers.  Also, why can't you just have this as a number itself in the raw field?